packages/flutter_markdown/test/link_test.dart

```diff
@@ -6,8 +6,12 @@ import 'package:flutter/gestures.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_markdown/flutter_markdown.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:markdown/markdown.dart' as md show version;
 import 'utils.dart';
 
+// TODO(Zhiguang): delete this once the min version of pkg:markdown is updated
+final bool _newMarkdown = md.version.compareTo('6.0.1') > 0;
```

Comment:
We have this in another test file, right? Maybe we should make it public and put it in a shared test file?

Reply:
Agreed, fixed!